incubator/hnc/pkg/controllers/suite_test.go

```diff
@@ -41,12 +42,17 @@ import (
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
 var (
-	cfg        *rest.Config
-	k8sClient  client.Client
-	testEnv    *envtest.Environment
-	k8sManager ctrl.Manager
+	cfg                 *rest.Config
+	k8sClient           client.Client
+	testEnv             *envtest.Environment
+	k8sManager          ctrl.Manager
+	newObjectController bool
 )
 
+func init() {
+	flag.BoolVar(&newObjectController, "enable-new-object-controller", false, "Enables new object controller.")
```

Comment:
I assume this is a temporary item until we fulling migrate everything to the new controller? Can we add some inline comments indicating the roadmap and some sort of target of when we can remove this as a boolean? The ticket or code don't really indicate the long term plan for this.

Reply:
Right - @yiqigao217 can you mention the GH issue?

Reply:
The GH issue is added in the comments.